SOURCES/0005-etc-rsyslog.d-xenserver.conf-mention-XCP-ng-in-addit.patch

```diff
@@ -0,0 +1,28 @@
+From f34cd5acbe9b8e7292783b843ad4d167bbb8a259 Mon Sep 17 00:00:00 2001
+From: Samuel Verschelde <stormi-xcp@ylix.fr>
+Date: Mon, 22 Sep 2025 18:32:10 +0200
+Subject: [PATCH 5/5] /etc/rsyslog.d/xenserver.conf: mention XCP-ng in addition
+ to XenServer
+
+I didn't completely replace XenServer with XCP-ng. After all, the file
+is still named xenserver.conf, so that would be awkward.
+
+Signed-off-by: Samuel Verschelde <stormi-xcp@ylix.fr>
+---
+ src/common/etc/rsyslog.d/xenserver.conf | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/common/etc/rsyslog.d/xenserver.conf b/src/common/etc/rsyslog.d/xenserver.conf
+index d59c633..0b65e65 100644
+--- a/src/common/etc/rsyslog.d/xenserver.conf
++++ b/src/common/etc/rsyslog.d/xenserver.conf
+@@ -1,5 +1,5 @@
+ # -- DO NOT EDIT THIS FILE --
+-# xenserver.conf contains the XenServer specific logging configurations
++# xenserver.conf contains the XenServer/XCP-ng specific logging configurations
+ # Any remote forwarding should be configured in another file in /etc/rsyslog.d/
+ # e.g. /etc/rsyslog.d/remote.conf
```

Comment:
Maybe make the statement more generic: any specific configuration should be configured in another file in /etc/rsyslog.d/